Alternative payments/Apple Pay Integration/IntegrateApplePay/Modules/ApplePay/JavaScript/ApplePay.View.js

```diff
@@ -0,0 +1,194 @@
+// @module DC.IntegrateApplePay.ApplePay
+define('DC.IntegrateApplePay.ApplePay.View', [
+	'dc_integrateapplepay_applepay.tpl',
+	'DC.IntegrateApplePay.ApplePay.Model',
+	'Backbone',
+	'LiveOrder.Model',
+	'jQuery'
+], function (
+	dc_integrateapplepay_applepay_tpl, ApplePayModel, Backbone, LiveOrderModel, jQuery
+) {
+	'use strict';
+
+	// @class DC.IntegrateApplePay.ApplePay.View @extends Backbone.View
+	return Backbone.View.extend({
+
+		template: dc_integrateapplepay_applepay_tpl,
+		initialize: function (options) {
+			this.container = options.container;
+			var checkout = this.container.getComponent("Checkout");
+			this.liveOrderModel = LiveOrderModel.getInstance();
+
+			var _this = this;
+			checkout.getCurrentStep().then(function (step) {
+				_this.stepInfo = step;
+			});
+			$.ajax({
+				url: "https://applepay.cdn-apple.com/jsapi/v1/apple-pay-sdk.js",
+				dataType: "script",
+				success: function () {
+					console.log("loaded");
+				}
+			});
+		},
+		events: {
+			'click #altapay-apple-pay-btn': 'applePayClicked'
+		},
+		applePayClicked: function () {
+			const self = this;
+			const request = {
+				"countryCode": this.container.getConfig().ApplePay.countryCode.toUpperCase() || "US",
+				"currencyCode": this.container.getConfig().ApplePay.currencyCode.toUpperCase() || "USD",
+				"merchantCapabilities": [
+					"supports3DS"
+				],
+				"supportedNetworks": [
+					"visa",
+					"masterCard"
+				],
```

Comment:
Check my last comments about adding it to the config, but fail validation when the config is saved with different values. If that is possible, then use the config instead.

Reply:
@emicha We can't put validation on saving because NetSuite does not provide such an option for the custom grid.